lang/src/main/antlr/PartiQL.g4

```diff
@@ -76,15 +81,38 @@ ddl
     ;
 
 createCommand
-    : CREATE TABLE symbolPrimitive                                                              # CreateTable
+    : CREATE TABLE tableName PAREN_LEFT tableDef PAREN_RIGHT                                    # CreateTable
     | CREATE INDEX ON symbolPrimitive PAREN_LEFT pathSimple ( COMMA pathSimple )* PAREN_RIGHT   # CreateIndex
     ;
 
 dropCommand
-    : DROP TABLE target=symbolPrimitive                         # DropTable
+    : DROP TABLE target=tableName                               # DropTable
     | DROP INDEX target=symbolPrimitive ON on=symbolPrimitive   # DropIndex
     ;
 
+tableDef
+    : tableDefPart ( COMMA tableDefPart )*
+    ;
+
+tableDefPart
+    : columnName type columnConstraint*                             # ColumnDeclaration
+    | ( CONSTRAINT tableConstraintName )?  tableConstraintDef       # TableConstraint
+    ;
+
+columnConstraint
+    : ( CONSTRAINT columnConstraintName )?  columnConstraintDef
+    ;
+
+columnConstraintDef
+    : NOT NULL                                  # ColConstrNotNull
+    | NULL                                      # ColConstrNull
+    | CHECK PAREN_LEFT expr PAREN_RIGHT         # ColConstrCheck
+    ;
+
+tableConstraintDef
+    : CHECK PAREN_LEFT expr PAREN_RIGHT         # TblConstrCheck
+    ;
```

Comment:
Following my previous comment about column constraints, I think a more relevant constraint to support as experimental for now is the concept of open/close? meaning additional attributes can get added to the table or not.

Reply:
IMO we know that we need to support both open and close schemas; no objection on leaving it out though.